codex-ui/src/vue/components/note-header/NoteHeader.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div
+    :class="$style['note-header']"
+    :style="{ '--opacity': props.opacity }"
```

Comment:
you can manage visibility of the header using `v-show` outside

Reply:
The v-show hides the component at all, once hidden it just disappears. Figma says it should stay, just be transparent.
![image](https://github.com/codex-team/notes.web/assets/94054053/f7eb17ef-eb23-4ab8-a6f3-6ed7e14d8b1b)

`Using v-show`
![image](https://github.com/codex-team/notes.web/assets/94054053/4a4f73d1-f3e9-49f7-835f-cc24c47afc45)

`Using opacity`
![image](https://github.com/codex-team/notes.web/assets/94054053/e23f82af-1798-4462-b781-a5e03dc1966f)